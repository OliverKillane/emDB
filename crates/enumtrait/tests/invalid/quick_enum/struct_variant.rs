#[enumtrait::quick_enum]
enum Invalid1 {
    Bing{ cool: i32 }
}

fn main() {}