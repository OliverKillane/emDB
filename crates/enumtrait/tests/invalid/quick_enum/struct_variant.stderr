error: Provided variants should be a single identifier of the type.

         = help: Try `enum Invalid1 { .. Bing, .. }`

 --> tests/invalid/quick_enum/struct_variant.rs:3:9
  |
3 |     Bing{ cool: i32 }
  |         ^^^^^^^^^^^^^
